Libraries/Lists/VirtualizedList.js

```diff
@@ -1331,9 +1331,21 @@ class VirtualizedList extends React.PureComponent<Props, State> {
     const preferredScrollerStyleDidChangeHandler =
       this.props.onPreferredScrollerStyleDidChange;
 
+    const isFirstRowSelected = this.state.selectedRowIndex === this.state.first;
+    const isLastRowSelected = this.state.selectedRowIndex === this.state.last;
+
+    // Don't pass in ArrowUp/ArrowDown at the top/bottom of the list so that keyboard event can bubble
+    let _validKeysDown = ['Home', 'End'];
```

Comment:
Why are Home and End always valid but ArrowUp and ArrowDown depend on which row is selected? Not blocking, just curious.

Reply:
Home and End scroll the scrollView, but don't move selection. So regardless of where you are in the list, they have an action they perform. I guess I could remove overriding `Home` if we're at the top of the list too, but I don't know that I have a way to check that like I can check which row is selected.